CONTRIBUTING.md

```diff
@@ -64,46 +78,18 @@ First, make sure you have the prerequisites installed and available on your `$PA
   - Rush will automatically manage the specific version needed by this repo as long as you have any v5 version installed.
   - If you're unable to install a global tool, you can instead call the wrapper script `node <repo root>/common/scripts/install-run-rush.js` any time the guide instructs you to run `rush`. The wrapper script will install a managed copy of Rush in a temporary directory for you.
 
-Next, get the code:
+### Building our repository
 
 1. Fork this repo
 2. Clone your fork locally (`git clone https://github.com/<youruser>/azure-sdk-for-js.git`)
 3. Open a terminal and move into your local copy (`cd azure-sdk-for-js`)
 4. Install and link all dependencies (`rush update`)
+5. Build the code base (`rush build`)
```

Comment:
should we make this `rush rebuild` just to future-proof it when folks re-run the command?

Reply:
hmm, what is wrong with incremental build the second time? both commands are described in the `Building` section.

Reply:
when I joined the team I was told to never use `rush build` because sometimes the incremental build doesn't detect changes and you'll pull your hair out, but maybe that's dated advice?

Reply:
That is not my experience so far but I can believe incremental build might cause some weird state if it does not track everything. I will change it to `rush rebuild`.